torch/hub.py

```diff
@@ -210,6 +216,71 @@ def _get_cache_or_reload(github, force_reload, verbose=True, skip_validation=Fal
     return repo_dir
 
 
+
+def _add_repo_to_trusted_list(repo, filepath, is_trusted=False):
+    # prompt
+    response = input(
+        f"The repository {repo} does not belong to the list of trusted repositories and as such cannot be downloaded. "
+        "Do you trust this repository and wish to add it to the trusted list of repositories (Y/n)?")
+    if response.lower() in {"y", 'yes'}:
+        if is_trusted:
+            print("The repository is already trusted.")
+            return
```

Comment:
I feel like we should just return without a message here. IIUC this should only ever happen if the user used `trust_repo=False` while the repo is already trusted?

Reply:
yes that's the case.
I thought that if one asks for `trust_repo=False` then answers `yes` to the prompt, then a notification that this was already trusted could be useful. But happy to remove this if there's no use.